src-electron/db/db-mapping.js

```diff
@@ -112,6 +112,7 @@ exports.map = {
         x.REPORTING_POLICY == dbEnums.reportingPolicy.suggested,
       reportingPolicy: x.REPORTING_POLICY,
       isSceneRequired: dbApi.fromDbBool(x.IS_SCENE_REQUIRED),
+      isFabricScoped: dbApi.fromDbBool(x.IS_FABRIC_SCOPED),
```

Comment:
Attributes do not have a concept of fabric scoping in this sense, so I'm not sure we should add this here.

What has such a concept is _structs_, and adding this for struct definitions would be useful.  Then we could get rid of the `struct_is_fabric_scoped` bit in helper-zcl.js and just condition that whole "figure out what the fabric index field and the fabric sensitive fields bits are" on the fabric-scoped boolean from the DB.